Broken link fixed (#17961)
kubernetes/website: content/en/docs/tasks/administer-cluster/access-cluster-api.md

}
```

> If the application is deployed as a Pod in the cluster, please refer to the [next section](#accessing-the-api-from-within-a-pod).

Reviewer: optional rewording:
```suggestion
If the application is deployed as a Pod in the cluster, see [Accessing the API from within a Pod](#accessing-the-api-from-within-a-pod).
```
